cernopendata/modules/fixtures/data/docs/delphi-guide-eventserver/delphi-guide-eventserver.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "body": {
+      "content": "delphi-guide-eventserver.md",
+      "format": "md"
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "This guide describes how to run the DELPHI stack from a container image."
+    },
+    "slug": "delphi-guide-eventserver",
+    "tags": [
+      "Getting Started"
+    ],
+    "title": "Running the DELPHI software stack using Eventserver",
```

Comment:
Why "Eventserver" and not "event server"?  The capitalisation seem to suggest that "Eventserver" might be some sort of a symbol name, but in the guide you mention that the name is "des". I would just use "event server" in the title, using casual English. (Ah, the term "Eventserver" might be inspired from German?)